src/tests/ftest/pool/RebuildTests.py

```diff
@@ -366,19 +282,18 @@ def test_multipool_rebuild(self):
             if pool1.pool_info.pi_rebuild_st.rs_errno != 0:
                 self.fail("P1 rebuild error reported: {}"
                           .format(pool1.pool_info.pi_rebuild_st.rs_errno))
-            if pool1.pool_info.pi_rebuild_st.rs_obj_nr != objcount:
+            if pool1.pool_info.pi_rebuild_st.rs_obj_nr != self.objcount:
                 self.fail("P1 rebuilt objs not as expected: {0} {1}"
                           .format(pool1.pool_info.pi_rebuild_st.rs_obj_nr,
-                                  objcount))
-            if pool1.pool_info.pi_rebuild_st.rs_rec_nr != (reccount*objcount):
+                                  self.objcount))
+            if pool1.pool_info.pi_rebuild_st.rs_rec_nr != (self.reccount*self.objcount):
```

Comment:
(pylint-line-too-long) Line too long (88/80)

Reply:
no belong to current patch.